gallery/gallery/pubspec.yaml

```diff
@@ -36,6 +36,7 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
+  grinder: ^0.8.0
```

Comment:
Does the grinder package have to be included in the gallery's pubspec.yaml? Since the gallery doesnt actually depend on it

Reply:
Yes, we need to be able to download the grinder tool.

It's just put in the `dev_dependencies` here so it won't be used when building the app.

If you think we should put it down one level (as said in the other comment) then I would need to create another pubspec.yaml one level below, which would solve your concern anyways

Reply:
+1. `dev_dependencies` is the recommended approach:

https://pub.dev/packages/grinder

You can also activate grinder globally and use `grind` from the command line, though that's optional.